build.sbt

```diff
@@ -2,10 +2,10 @@ import com.typesafe.sbt.SbtGhPages.ghpages
 import com.typesafe.sbt.SbtGit.git
 import de.heikoseeberger.sbtheader.HeaderPattern
 
-lazy val buildSettings = Seq(
+lazy val commonSettings = Seq(
   organization := "com.fortysevendeg",
```

Comment:
Starting now and going forward we are gonna use `com.47deg` as the org name and groupId for all libs going forward. Existing libs will be migrated over as next releases happen to bring all the OS 47 libs under the same groupId. Once you are ready to publish to sonatype and central, let @fedefernandez and I know and we'll let you when ready. We need to request the org with the first lib that makes there. /cc @andyscott